.gitattributes

```diff
@@ -1,4 +1,5 @@
 * text=auto
+*.js eol=lf
```

Comment:
This is needed to get around an issue with CRLF I'm having on windows. Will investigate this a bit more and might create a new issue to address it.

Reply:
Sorry, we don't need, it will break tests cases, avoid it adding here

Reply:
Yea I removed that. Now developing on a linux machine to make life easier.